src/app/project/project.component.html

```diff
@@ -24,7 +24,26 @@ <h1>{{getProjectName()}}</h1>
       </div>
       <mat-tab [label]="tabConfig[1].label">
         <div class="vf-u-margin__top--xxl">
-          <app-template-questionnaire></app-template-questionnaire>
+          <details class="vf-details vf-box vf-box--outline" [attr.open]="getProjectGeoAccession()">
+            <summary class="vf-details--summary">Download Spreadsheet Using GEO Accession</summary>
+            <div class="vf-u-margin__top--xxl">
+              <p *ngIf="getProjectGeoAccession()">Download spreadsheet using GEO accession {{getProjectGeoAccession()}}</p>
```

Comment:
Have you considered making this download button a component?

Reply:
^ @yusra-haider